kalite/distributed/static/js/distributed/exercises/api_resources.py

```diff
@@ -0,0 +1,31 @@
+from django.db import models
+from django.http import HttpResponse
+
+from tastypie import fields
+from tastypie.resources import ModelResource
+
+import yaml
```

Comment:
We discussed this in the dev room. We're gonna make a function/command that turns all yml files into json files, and then read those json files when they're available.